framework/src/utils/RankTwoTensor.C

```diff
@@ -93,6 +93,24 @@ RankTwoTensor::RankTwoTensor(const TypeVector<Real> & row1,
     _vals[two_n + i] = row3(i);
 }
 
+RankTwoTensor
+RankTwoTensor::initializeFromRows(const TypeVector<Real> & row1,
```

Comment:
I was keeping consistency with the existing constructor, but since I suggested we deprecate that, I might as well rename the parameters.